docs/topics/DatabaseOperations.adoc

```diff
@@ -58,6 +58,19 @@ image::unlock_database.png[]
 .Unlocked database
 image::database_view.png[]
 
+=== Quick Unlock
+On Windows and macOS, depending on hardware availability, KeePassXC will securely store your credentials to enable quickly unlocking your database after providing biometric authentication. This is enabled by default on Windows using _Windows Hello_ and on macOS using _Touch ID or Apple Watch_ services. You can disable this feature in the Application Settings under the Security section.

```

Comment:
Should we say "KeePassXC will securely store your credentials" when in fact we are just asking the operating system to do so meaning the _security_ is provided by the OS and not us? Splitting hairs but noting.

Reply:
Maybe "Your credentials will be securely stored ..."